upgrading/automated_upgrades.adoc

```diff
@@ -286,15 +286,19 @@ requirements.
 .. Back up the following files that would be required should you later need to
 downgrade back to {product-title} 3.9:
 
-... On master hosts, back up the following, including the *systemd* files that will
-be removed during the upgrade to the control plane static pod architecture:
+... On master hosts, back up the following files:
 +
 ----
 /usr/lib/systemd/system/atomic-openshift-master-api.service
 /usr/lib/systemd/system/atomic-openshift-master-controllers.service
 /etc/sysconfig/atomic-openshift-master-api
 /etc/sysconfig/atomic-openshift-master-controllers
+/etc/origin/master/master-config.yaml
+/etc/origin/master/scheduler.json
 ----
++
+The *systemd* files are removed during the upgrade to the control plane static
+pod architecture.
```

Comment:
@kalexand-rh Does this mean:

"The systemd files that are removed during the upgrade are sent to the control plane static pod architecture."

or

"The systemd files are removed when you perform the upgrade to the control plane static pod architecture."

?